spec/controllers/posts_controller_spec.rb

```diff
@@ -89,5 +90,39 @@
       end
     end
 
+    describe "#create" do
```

Comment:
Very nice test! @schaui6 : thoughts on if this is digestible enough for students? Relatedly, should we include bitly gem or have the students figure that out?